commands/__init__.py

```diff
@@ -184,6 +184,22 @@ class Columns(Enum):
     default=True,
     help="Check constraints before deployments, e.g. environment restrictions",
 )
+@click.option(
+    "-en",
```

Comment:
I would remove these shortcuts. In my opinion it is not clear enough and I would use the long name.

Reply:
I agree.